src/utils/businessHours.test.ts

```diff
@@ -487,4 +488,72 @@ describe('businessHours tests', function () {
       );
     });
   });
+
+  describe('getBusinessHoursLeft', function () {
+    it('should correctly calculate business hours left overnight', function () {
+      const triageBy = '2023-12-22T18:00:00.000Z';
+      const now = moment('2023-12-22T01:00:00.000Z');
+      expect(
+        getBusinessHoursLeft(
+          triageBy,
+          now,
+          'test-ttt-simple',
+          'getsentry',
+          'Other'
+        )
+      ).toEqual(1);
+    });
+    it('should correctly calculate business hours over multiple days', function () {
+      const triageBy = '2023-12-22T18:00:00.000Z';
+      const now = moment('2023-12-21T01:00:00.000Z');
+      expect(
+        getBusinessHoursLeft(
+          triageBy,
+          now,
+          'test-ttt-simple',
+          'getsentry',
+          'Other'
+        )
+      ).toEqual(9);
+    });
+    it('should correctly account for weekends when calculating business hours', function () {
+      const triageBy = '2023-01-17T18:00:00.000Z';
```

Comment:
Just curious what happens when we don't include milliseconds (or do include microseconds) in an ISO8601 timestamp. I know that Python's deserializer sometimes barfed on that, which caused trouble and test flakes in the past. Maybe JS's is smarter though.

Reply:
this will be fine, since the moment library will parse the timestamp without milliseconds properly